getsavvyinc/upgrade-cli
# upgrade: add tests for asset, checksum pkgs

- checksum: empty checksum file should return an error
- checksum: test checksum validator
- checksum_test: refactor setupTestServer to accept a http.Handler
- create separate checksum pkg
- create separate release pkg
- release,checksum: rename types to avoid pkg stutter
- download: verify downloading assets doesn't change the content
- refactor: mv download*.go to its own asset pkg
